src/components/InQueueMessaging/InQueueVoicemailComponent.js

```diff
@@ -296,11 +296,6 @@ class InQueueVoicemailComponent extends React.Component {
             multiline
             rows='4'
             fullWidth
-            InputProps={{
```

Comment:
@ktalebian since you're my React.js guru, I did want to follow up on this. Why is this considered invalid React? `InputProps` seems to be a valid object according to https://material-ui.com/api/text-field/.

Reply:
It looks like it should be `inputProps` not `InputProps` from the docs you linked.